src/OSPSuite.InstallationValidator.Core/Reporting/BatchComparisonResultReporter.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using OSPSuite.Core.Domain;
+using OSPSuite.Infrastructure.Reporting;
+using OSPSuite.InstallationValidator.Core.Assets;
+using OSPSuite.InstallationValidator.Core.Domain;
+using OSPSuite.TeXReporting.Builder;
+using OSPSuite.TeXReporting.Items;
+
+namespace OSPSuite.InstallationValidator.Core.Reporting
+{
+   public class BatchComparisonResultBuilder : OSPSuiteTeXBuilder<BatchComparisonResult>
+   {
+      private readonly ITeXBuilderRepository _teXBuilderRepository;
+
+      public BatchComparisonResultBuilder(ITeXBuilderRepository teXBuilderRepository)
+      {
+         _teXBuilderRepository = teXBuilderRepository;
+      }
+
+      public override void Build(BatchComparisonResult comparisonResult, OSPSuiteTracker buildTracker)
+      {
+         var objectsToReport = new List<object>
+         {
+            new Section(Assets.Reporting.BatchComparisonResults),
+            new SubSection(Assets.Reporting.ComparisonFolders), firstComparisonFolder(comparisonResult), new LineBreak(), secondComparisonFolder(comparisonResult),
+            new SubSection(Assets.Reporting.DefaultTolerances), tolerancesFor(comparisonResult),
+            new SubSection(Assets.Reporting.OverallComparisonResult), validationResultFor(comparisonResult)
+         };
+
+         var fileComparisonResults = comparisonResult.FileComparisonResults.Where(x => x.State != ValidationState.Valid).ToList();
+         if(fileComparisonResults.Any())
+            objectsToReport.Add(new SubSection(Assets.Reporting.FailedValidations));
+         objectsToReport.AddRange(fileComparisonResults);
+
+         _teXBuilderRepository.Report(objectsToReport, buildTracker);
+      }
+
+      private ValidationState validationResultFor(BatchComparisonResult comparisonResult)
+      {
+         return comparisonResult.State;
+      }
+
+      private object tolerancesFor(BatchComparisonResult comparisonResult)
+      {
+         return "todo";
```

Comment:
This has been added to Core. I will update PKSim batch tomorrow to use those variables and then Installation should be able to read them. I don;t think we will find a global value tho. This will be defined PER Simulation and probably will be reported for each failing simulations